edb/server/cluster.py

```diff
@@ -418,6 +421,38 @@ async def _new_pg_cluster(self) -> pgcluster.Cluster:
     def get_data_dir(self) -> pathlib.Path:
         return self._data_dir
 
+    def get_runstate_dir(self) -> pathlib.Path:
+        return self._runstate_dir
+
+    def get_jws_key(self) -> jwk.JWK:
+        if self._jws_key is None:
+            self._jws_key = self._load_jws_key()
+        return self._jws_key
+
+    def _load_jws_key(self) -> jwk.JWK:
+        jws_key_file = self._get_jws_key_path()
+        try:
+            with open(jws_key_file, 'rb') as kf:
+                jws_key = jwk.JWK.from_pem(kf.read())
+        except Exception as e:
+            raise ClusterError(f"cannot load JWS key: {e}") from e
+
+        if (
+            not jws_key.has_public
+            or jws_key['kty'] not in {"RSA", "EC"}
```

Comment:
Just curious, why we have two kinds of keys supported?

Reply:
Historical reasons, but we mostly settled on EC now, so I guess we can remove the RSA option.